libaleth-interpreter/VM.h

```diff
@@ -17,9 +17,10 @@
 
 #pragma once
 
-#include "Instruction.h"
 #include "VMConfig.h"
-#include "VMFace.h"
+
+#include <libevm/VMFace.h>
```

Comment:
So libaleth-interpreter depends on libevm? This is going to change later, right?

Reply:
Only on headers. This should change, but is not a priority. Home helper functions are defined in VMFace.h. I didn't want to do a lot of changes when moving files.

At least this PR allow building aleth-interpreter as a shared library and distribute it in this form.